spanner/crud.js

```diff
@@ -163,6 +163,50 @@ function readData (instanceId, databaseId) {
   // [END read_data]
 }
 
+function readStaleData (instanceId, databaseId) {
+  // [START read_stale_data]
+  // Imports the Google Cloud client library
+  const Spanner = require('@google-cloud/spanner');
+
+  // Instantiates a client
+  const spanner = Spanner();
+
+  // Uncomment these lines to specify the instance and database to use
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Gets a reference to a Cloud Spanner instance and database
+  const instance = spanner.instance(instanceId);
+  const database = instance.database(databaseId);
+
+  // Read rows from the Albums table
+  const albumsTable = database.table('Albums');
+
+  const query = {
+    columns: ['SingerId', 'AlbumId', 'AlbumTitle', 'MarketingBudget'],
+    keySet: {
+      all: true
+    }
+  };
+
+  const options = {
+    // Guarantees that all writes that have committed more than 10 seconds ago
```

Comment:
Nit: remove `that have` and combine into one line

Reply:
Done